packages/react-server/src/ReactServerStreamConfigBun.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+type BunReadableStreamController = ReadableStreamController & {
+  end(): mixed,
+  write(data: Chunk): void,
+  error(error: Error): void,
+};
+export type Destination = BunReadableStreamController;
+
+export type PrecomputedChunk = string;
+export opaque type Chunk = string;
+
+export function scheduleWork(callback: () => void) {
+  callback();
+}
+
+export function flushBuffered(destination: Destination) {
+  // WHATWG Streams do not yet have a way to flush the underlying
+  // transform streams. https://github.com/whatwg/streams/issues/960
+}
+
+// For now we support AsyncLocalStorage as a global for the "browser" builds
+// TODO: Move this to some special WinterCG build.
+// export const supportsRequestStorage = typeof AsyncLocalStorage === 'function';
+// export const requestStorage: AsyncLocalStorage<
+//   Map<Function, mixed>,
+// > = supportsRequestStorage ? new AsyncLocalStorage() : (null: any);
```

Comment:
is AsyncLocalStorage on the roadmap? I would generally say just delete the comments since they can be referenced from other implementations later if you add this back in

Reply:
Not in the near term. I'll delete those comments - good call